dasharo-security/me-keyword.resource

```diff
@@ -0,0 +1,9 @@
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check If ME Is Disabled
+    ${out}=    Run Process    /usr/sbin/setpci    -s    16.0    40.L
```

Comment:
Run Process executes program on your computer (where the `robot`) process is started, you should use `Execute Command In Terminal` instead, to run on the DUT

Reply:
fixed